src/main/java/de/srendi/advancedperipherals/common/data/ItemTagsProvider.java

```diff
@@ -19,5 +19,9 @@ protected ItemTagsProvider(DataGenerator generator, @Nullable ExistingFileHelper
     @Override
     protected void addTags() {
         tag(APTags.Items.SMART_GLASSES).add(APItems.SMART_GLASSES.get()).add(APItems.SMART_GLASSES_NETHERITE.get());
+        tag(de.srendi.advancedperipherals.common.addons.ae2.Integration.getCableP2PTag())
```

Comment:
Is there a reason why you use the full qualifier here and don't import it?

Reply:
> Is there a reason why you use the full qualifier here and don't import it?

Because `Integration` is defined in multiple packages. In order to not mix them up, I decide use full path here.